views/partials/profileContent.handlebars

```diff
@@ -48,9 +50,23 @@
                                 <div class="">
                                     <h3 class="user-info is-size-3 has-text-centered b-bot">Your items</h3>
 
+                                    {{!-- Update to make look better --}}
```

Comment:
Can you clarify?

Reply:
I didn't really put effort into making the products on the /profile page look nice - I can make it look nice tomorrow, or someone else can.